tests/integration/test_acme_tls/test.py

```diff
@@ -0,0 +1,161 @@
+import logging
```

Comment:
Why not Pebble?

https://github.com/letsencrypt/Pebble

Reply:
Thank you, I was looking at it at some point and honestly forgot about it; let's try to use it in tests.

One thing that might be tricky is hooking functions to add counters for the tests.

Reply:
There are some additional api to track the internal status, it could be useful for that